src/frontend/ts/state/chessStore.ts

```diff
@@ -131,18 +119,30 @@ const useChessStore = create<ChessState>((set, get) => ({
       case Update.Result: {
         const resultData = data as ResultUpdate
         const gameID = resultData.gameID
+        const currentGame = get().activeGames.get(gameID)
+        const archivedGame = get().archivedGames.get(gameID)
```

Comment:
Please add a comment above this line that this game should already exist in the archive, based on the ordering of cards coming from `%chess`